docs/dev-guides/AutomatedTestswithFitnesseSetup/index.md

```diff
@@ -0,0 +1,295 @@
+
+## Running fitnesse from eclipse {#running_fitnesse_from_eclipse}
+
+-   Prerequisiste is an installed development environment.
+-   Get the \"fitnesse\" java project which fits to your release into
+    your eclipse workspace. You will find the sources on github.
+    <https://github.com/idempiere/idempiere-fitnesse>
+-   following folder structure is necessary for iDempiere Dev and the
+    fitnesse part (github standards must be renamed!):
+    -   idempiere
+    -   idempiere-fitnesse
+        -   org.idempiere.fitnesse.\*
+        -   org.idempiere.fitrecorder
+        -   org.idempiere.ui.zk.selenium
+-   Before importing the Fitnesse-plugin into the iDempiere Workspace
+    you must run "mvn verify" in the idempiere-fitnesse folder !
+    -   To avoid Tycho/Maven problems, remove
+        \~/.m2/repository/.meta/p2-artifacts.properties prior to running
+        maven build for source using Tycho 2.6 and below, i.e. with: rm
+        \~/.m2/repository/.meta/p2-artifacts.properties (rm \~ is
+        helpful cause the file is difficult to find otherwise)
+-   Import - Existing Projects into Workspace + there tick "Search for
+    nested Projects" (see image below)
+-   Set ADEMPIERE_WEB_PORT StringSubstitution to right Port with SSL
+    e.g. 8443 (older versions work with 8080)
+    -   The value is set in Eclipse - Preferences - Run/Debug - String
+        Substitution
+    -   The former port mentioned in older and following screenshots
+        does not work anymore as SSL is mandatory for iDempiere.
+    -   Use 8443 or your selected SSL-Port for iDempiere Access in setup
+        for String Substitution !!!:
+
+![Screen from fitnesse](./Adempiere_Web_Port_Adaption_2.png)
+
+
+## Run IDempiere Fitnesse Bundles {#run_idempiere_fitnesse_bundles}
+
+-   First you have to define the Fitnesse Application in the Eclipse -
+    Configurations
+
+![Fitnesse configuration in Eclipse](./Eclipse_configuration_fitnesse_component.png)
+
+-   Use \"Run\" for starting the fitnesse component. Debug was required
+    in older versions.
+-   Verify fitnesse started ok.
+
+![Fitnesse-started.png](./Fitnesse-started.png)
+
+-   Please start the bundles at the idempiere server\'s osgi as shown
+    below in the screenshot:
+
+![Bundles.png](./Bundles.png)
+
+### Run IDempiere Fitnesse Bundles {#run_idempiere_fitnesse_bundles_1}
+
+Congratulations finished! Now you can start using Fitnesse with
+iDempiere e.g. start some of the TestCases already prepared for Login or
+a new Business partner.
+
+Some special cases are:
+
+-   If your server is running with a self-signed certificate, you must
+    match the certificates in the respective cacert and key.pem. You
+    find some commands to do this on one computer containing both
+    iDempiere and Fitnesse-server below: [Installing
+    Fitnesse.HowTo#Allow SSL-Connection for self signed
+    certificates](Installing_Fitnesse.HowTo#Allow_SSL-Connection_for_self_signed_certificates_: "wikilink")
```

Comment:
you missing file (Installing_Fitnesse.HowTo) on somewhere

Reply:
Thank you for your valuable comments. Actually I deleted not all parts I wanted from fitnesse etc. 
Now I looked through it and hope I got it all. The internal test by git and docusaurus also went well.
The main files I created are in the area of **basic functional** on login and menue.